src/tslint-cli.ts

```diff
@@ -254,11 +254,11 @@ const processFiles = (files: string[], program?: ts.Program) => {
 
     const lintResult = linter.getResult();
 
-    if (lintResult.failureCount > 0) {
-        outputStream.write(lintResult.output, () => {
+    outputStream.write(lintResult.output, () => {
+        if (lintResult.failureCount > 0) {
             process.exit(argv.force ? 0 : 2);
-        });
-    }
+        }
+    });
```

Comment:
Strictly speaking, this is out of scope, but I adopted this change from PR #1463 anyway.